rust-query-crlite/Cargo.toml

```diff
@@ -7,15 +7,15 @@ edition = "2021"
 base64 = "0.13"
 bincode = "1.3"
 byteorder = "1.2.7"
-clap = { version = "3.0", features = ["derive"] }
+clap = { version = "3.2", features = ["derive"] }
 der-parser = "9.0"
 hex = "0.4"
 log = "0.4"
 num-bigint = "0.4"
 pem = "1.0"
 reqwest = { version = "0.11", features = ["blocking", "json", "rustls-tls"] }
 rust_cascade = "1.4.0"
-rustls = "0.21"
+rustls = { version = "0.21", features = ["dangerous_configuration"] }
```

Comment:
Why the `dangerous_configuration`?

Reply:
In version 0.21.x of rustls (sometime after I first wrote this but before the last 0.21.x release), rustls gated access to the `rustls::client::ServerCertVerifier` trait behind the `dangrous_configuration` feature. The `rust-query-crlite https <url>` command uses a `ServerCertVerifier` that accepts all certs so that it can reliably extract certificates from TLS handshakes for revocation checking. It's useful if, say, the server presents a certificate with a BR violation that would warrant revocation.